lib/versioning.h

```diff
@@ -150,9 +150,9 @@
 
 
 /* config version code, in the same format as GlobalConfig->version */
-#define VERSION_VALUE   0x0318
-#define VERSION_CURRENT VERSION_3_24
-#define VERSION_CURRENT_VER_ONLY "3.24"
+#define VERSION_VALUE   0x0319
+#define VERSION_CURRENT VERSION_3_25
```

Comment:
@furiel needed `VERSION_3_25` and `VERSION_VALUE_3_25` in a previous PR, so that's already added :+1: .